docs/TUTORIAL.md

```diff
@@ -1,3 +1,163 @@
 # Tutorial
 
-Coming soon!
+# Tutorial
+
+The **ksonnet** [readme][readme] 
+shows you how to create a default 
+`deployment.json` file that lets you deploy an nginx container 
+to an existing Kubernetes cluster. This tutorial shows you how to:
+
+* Modify the deployment using **ksonnet** definitions
+* Define other Kubernetes objects
+* Work with mixins to develop complex configurations
+
+## Prerequisites
+
+This tutorial assumes that you have performed the following 
+tasks. For details, see the [readme][readme].
+
+* Installed **Jsonnet**
+* Cloned the **ksonnet** repository locally
+* Installed and configured the VisualStudio Code extension 
+(optional)
+* Created a test Kubernetes cluster
+
+NOTE: All import paths are relative to the root of the 
```

Comment:
Backed this one out, sort of. Added step to put tutorial files at root of ksonnet repo. Rationale: this way reader can copy bash literally without having to include path. Added note about -J flag and path if file is elsewhere.